cfpb/consumerfinance.gov
# Breadcrumbs: Remove horizontal scroll

The [original breadcrumb component](https://github.com/cfpb/consumerfinance.gov/commit/75c3fc9ae0336e6b90ce3c6d9b1ba974ac68df03) doesn't wrap and adds a horizontal scroll at small screen sizes. Discussion in el-camino chat has decided to make it wrap so all breadcrumbs are visible and we'll revisit breadcrumb small screen behavior more in the future.

Fixes https://[GHE]/CFGOV/platform/issues/4025

## Changes

- Remove horizontal scroll from breadcrumbs and wrap them instead when at small screen sizes.

## How to test this PR

The following pages—if they have breadcrumbs—should wrap when the page is too narrow (compare to horizontal scrollbar on production). When tabbing to the breadcrumb the focus rectangle should have four sides (compare to missing bottom outline on production):

http://localhost:8000/about-us/diversity-and-inclusion/voluntary-assessment-onboarding-form/
http://localhost:8000/about-us/diversity-and-inclusion/voluntary-assessment-onboarding-form/form-submitted/
http://localhost:8000/about-us/diversity-and-inclusion/voluntary-assessment-onboarding-form/privacy-act-statement/

http://localhost:8000/data-research/hmda/historic-data/

http://localhost:8000/ask-cfpb/what-is-a-credit-reporting-company-en-1251/
http://localhost:8000/ask-cfpb/search/?q=test

http://localhost:8000/about-us/blog/borrowing-or-lending-money-to-family-or-friends-use-this-worksheet/
http://localhost:8000/compliance/supervision-examinations/compliance-management-review-examination-procedures/
http://localhost:8000/enforcement/actions/capital-one-bank/

http://localhost:8000/about-us/events/archive-past-events/2021-release-of-the-taskforce-on-federal-consumer-financial-law-report/

http://localhost:8000/compliance/compliance-resources/consumer-lending-resources/payday-lending-rule/payday-lending-rule-faqs/

http://localhost:8000/about-us/careers/current-openings/examiner-commissioned-compliance-examiner-21-cfpb-158-mpa-/ (or another open position if that one is gone) 

http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/
http://localhost:8000/consumer-tools/educator-tools/youth-financial-education/teach/activities/sonya-gives-advice/

http://localhost:8000/data-research/prepaid-accounts/search-agreements/detail/72669/

http://localhost:8000/consumer-tools/financial-well-being/results/
http://localhost:8000/consumer-tools/financial-well-being/results/es/

http://localhost:8000/consumer-tools/financial-well-being/error/
http://localhost:8000/consumer-tools/financial-well-being/error/es/
http://localhost:8000/consumer-tools/financial-well-being/about/
http://localhost:8000/consumer-tools/financial-well-being/about/es/
